*** boost_1_45_0/tools/build/v2/build/configure.jam.org	2011-10-26 12:18:07.820255008 +0800
--- boost_1_45_0/tools/build/v2/build/configure.jam	2011-10-26 12:24:02.068255005 +0800
***************
*** 140,157 ****
          }
          else 
          {           
-             local x = [ PAD "    - $(what)" : $(.width) ] ;            
-             if [ UPDATE_NOW $(jam-targets) :
-                  $(.log-fd) : ignore-minus-n : ignore-minus-q ] 
-             {
-                 .$(what)-supported.$(ps) = yes ;
-                 result = true ;
-                 log-check-result "$(x) : yes" ;
-             }        
-             else
-             {
-                 log-check-result "$(x) : no" ;
-             }
          }        
          return $(result) ;
      }    
--- 140,145 ----
