*** soprano-v2.7.0/cmake/modules/FindRedland.cmake.org	2011-10-27 09:30:56.892255007 +0800
--- soprano-v2.7.0/cmake/modules/FindRedland.cmake	2011-10-27 09:37:56.816255007 +0800
***************
*** 43,48 ****
--- 43,60 ----
      # extract include paths from redland-config
      execute_process(
        COMMAND ${REDLAND_CONFIG_EXECUTABLE} --cflags
+       OUTPUT_VARIABLE redland_CFLAGS_ARGS)
+     string( REPLACE " " ";" redland_CFLAGS_ARGS ${redland_CFLAGS_ARGS} )
+     foreach( _ARG ${redland_CFLAGS_ARGS} )
+       if(${_ARG} MATCHES "^-L")
+         string(REGEX REPLACE "^-L" "" _ARG ${_ARG})
+         list(APPEND redland_LIBRARY_DIRS ${_ARG})
+       endif(${_ARG} MATCHES "^-L")
+     endforeach(_ARG)
+ 
+     # extract lib paths from redland-config
+     execute_process(
+       COMMAND ${REDLAND_CONFIG_EXECUTABLE} --libs
        OUTPUT_VARIABLE redland_LIBS_ARGS)
      string( REPLACE " " ";" redland_LIBS_ARGS ${redland_LIBS_ARGS} )
      foreach( _ARG ${redland_LIBS_ARGS} )
***************
*** 53,69 ****
        endif(${_ARG} MATCHES "^-I")
      endforeach(_ARG)
  
-     # extract lib paths from redland-config
-     execute_process(
-       COMMAND ${REDLAND_CONFIG_EXECUTABLE} --libs
-       OUTPUT_VARIABLE redland_CFLAGS_ARGS)
-     string( REPLACE " " ";" redland_CFLAGS_ARGS ${redland_CFLAGS_ARGS} )
-     foreach( _ARG ${redland_CFLAGS_ARGS} )
-       if(${_ARG} MATCHES "^-L")
-         string(REGEX REPLACE "^-L" "" _ARG ${_ARG})
-         list(APPEND redland_LIBRARY_DIRS ${_ARG})
-       endif(${_ARG} MATCHES "^-L")
-     endforeach(_ARG)
    endif(REDLAND_VERSION)
  endif(REDLAND_CONFIG_EXECUTABLE)
  
--- 65,70 ----
***************
*** 72,85 ****
  
  
  find_path(REDLAND_INCLUDE_DIR redland.h
!   HINTS
    ${redland_INCLUDE_DIRS}
    /usr/X11/include
    PATH_SUFFIXES redland
    )
  
  find_library(REDLAND_LIBRARIES NAMES rdf librdf
!   HINTS
    ${redland_LIBRARY_DIRS}
    )
  
--- 73,86 ----
  
  
  find_path(REDLAND_INCLUDE_DIR redland.h
!   PATHS
    ${redland_INCLUDE_DIRS}
    /usr/X11/include
    PATH_SUFFIXES redland
    )
  
  find_library(REDLAND_LIBRARIES NAMES rdf librdf
!   PATHS
    ${redland_LIBRARY_DIRS}
    )
  
***************
*** 105,122 ****
    set(CMAKE_FIND_LIBRARY_SUFFIXES ${_SUFFIXES})
    if(_REDLAND_STORAGE_LIBS)
      message(STATUS "Found Redland storage: ${_REDLAND_STORAGE_LIBS}")
-     try_run(_TEST_EXITCODE _TEST_COMPILED
-       "${CMAKE_CURRENT_BINARY_DIR}"
-       "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/CheckLibraryLoads.c"
-       CMAKE_FLAGS "-DLINK_LIBRARIES=dl"
-       RUN_OUTPUT_VARIABLE _OUTPUT
-       ARGS ${_REDLAND_STORAGE_LIBS}
-     )
-     if(NOT "${_TEST_EXITCODE}" EQUAL 0)
-       set(_REDLAND_VERSION_OK)
-       message(STATUS "${_OUTPUT}")
-       message(STATUS "Redland with broken NEEDED section detected, disabling")
-     endif(NOT "${_TEST_EXITCODE}" EQUAL 0)
    endif(_REDLAND_STORAGE_LIBS)
  
  endif(NOT WIN32)
--- 106,111 ----
